### PR TITLE
DOC: Add documentation for DecompositionView

### PR DESCRIPTION
**Pull request by ElDeveloper**

### CODE REVIEW

File: emperor/support_files/js/view.js [hunk 1]
```diff
@@ -6,44 +6,69 @@ define([
...
+ *
+ */
 function DecompositionView(decomp) {
   /* The length of this list attributes is len(DecompositionModel.ids) */
 
+  /**
+   * The decomposition model that the view represents.
+   * @type {DecompositionModel}
+   */
   this.decomp = decomp; // The decomposition model seen by the view
```
Comment (squirrelo) [inline emperor/support_files/js/view.js]: Could you remove the comment from this line? The info above is probably enough to explain the variable.

File: emperor/support_files/js/view.js [hunk 2]
```diff
@@ -6,44 +6,69 @@ define([
...
+  /**
+   * Number of visible samples.
+   * @type {integer}
+   */
   this.visibleCount = this.count;
+  /**
+   * Top visible dimensions
+   * @type {integer[]}
+   */
   this.visibleDimensions = [0, 1, 2]; // We default to the first three PCs
```
Comment (squirrelo) [inline emperor/support_files/js/view.js]: Could you remove this comment and add it as `@default` to the above info?

File: emperor/support_files/js/view.js [hunk 3]
```diff
@@ -6,44 +6,69 @@ define([
...
 
   this.tubes = []; // Three.meshes
```
Comment (squirrelo) [inline emperor/support_files/js/view.js]: Do these need documentation?

File: emperor/support_files/js/view.js [hunk 4]
```diff
@@ -6,44 +6,69 @@ define([
...
 
   // setup this.markers and this.lines
   this._initBaseView();
+
+  /**
+   * True when changes have occured that require re-rendering of the canvas
+   * @type {boolean}
+   */
   this.needsUpdate = true;
   // this.elementOrdering = []; // list of ints - Not sure if needed
```
Comment (squirrelo) [inline emperor/support_files/js/view.js]: Can you remove this line?

File: emperor/support_files/js/view.js [hunk 5]
```diff
@@ -140,7 +168,14 @@ DecompositionView.prototype.setCategory = function(attributes,
   return dataView;
 };
 
-/* Change the color for a set of plottables - group: list of plottables */
+/**
+ *
+ * Change the color for a set of plottables.
+ *
+ * @param {integer} color An RGB color in hexadecimal format.
+ * @param {Plottable[]} plottables Array of objects that will change in color.
```
Comment (squirrelo) [inline emperor/support_files/js/view.js]: plottables -> group. Also, is this accurate?

**Comment by squirrelo** [comment]
Few comments, but looks pretty good!

**Comment by ElDeveloper** [comment]
Thanks for the review, all comments should be addressed now.

**Pull request merged**